__tests__/app.test.js

```diff
@@ -42,3 +43,61 @@ describe("Invalid path", () => {
   });
 }); // end of invalid path testing
 
+describe("/api", () => {
+
+  test("respond with 200 status code", () => {
+   return request(app).get("/api").expect(200);
+  });
+
+  test("respond with object listing all available endpoints", () => {
+    return request(app)
+     .get("/api")
+     .then((data) => {
+      expect(data.body.fetchEndPoints).toEqual(JSONendpoints);
+     });
+   });
+
+}); // end of "/api/topics" testing
+
+describe("/api/articles/:article_id", () => {
+
+  test("should respond with the correct status code", () => {
+    return request(app).get("/api/articles/2").expect(200);
+  })
+
+  test("responds with article object with matching id", () => {
+    return request(app)
```

Comment:
No need to do separate requests for .expect(200) and checking what comes back. Can just add a .then. 

What you have done won't break or anything though :)